src/hooks/UseToggle.jsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+function UseToggle() {
+  const [Infor, setInfor] = useState([]);
```

Comment:
When you create a new state, make sure you follow the naming conventions.
the value and setter should be in camel case:
```js
const [infor, setInfor] = useState([]);
```